code/modules/mob/living/carbon/brain/MMI.dm

```diff
@@ -300,3 +299,41 @@
 
 	. = ..()
 	brainmob.update_runechat_msg_location()
+
+
+/obj/item/mmi/syndie
+	name = "\improper Syndicate Man-Machine Interface"
+	desc = "Syndicate's own brand of MMI. Mindslaves any brain inserted into it for as long as it's in. Cyborgs made with this MMI will be slaved to the owner. Does not fit into NT AI cores."
+	origin_tech = "biotech=4;programming=4;syndicate=2"
+	syndiemmi = TRUE
+	mmi_item_name = "Syndicate Man-Machine Interface"
+
+/obj/item/mmi/syndie/attackby(obj/item/O, mob/user, params)
+	if(!master_uid && ishuman(user) && istype(O,/obj/item/organ/internal/brain))
+		to_chat(user, "<span class='notice'>You press your thumb on [src] and imprint your user information.</span>")
+		master_uid = user.UID()
+		if(!user.mind.has_antag_datum(/datum/antagonist/traitor))
+			message_admins("[user] has mindslaved [O] using a Syndicate MMI, but they are not a traitor!")
+	..()
+
+/obj/item/mmi/syndie/become_occupied(new_icon)
+	..()
+	brainmob.mind.remove_antag_datum(/datum/antagonist/mindslave) //Overrides any previous mindslaving
+
+	if(master_uid)
+		var/mob/living/carbon/human/master = locateUID(master_uid)
+
+		if(master)
+			to_chat(brainmob, "<span class='userdanger'>You feel the MMI overriding your free will!</span>")
+			brainmob.mind.add_antag_datum(new /datum/antagonist/mindslave(master.mind))
+			return
```

Comment:
This stops working when the master *body* gets gibbed, even if their brain survives (and gets cloned/mmi'd later)
Should you maybe store the *mind* as a master? Those tend to be a bit more robust to the rigors of space. Not that you are using master's mob for anything...

Reply:
Will look into that now, I had the MMI store the mob because I wasnt sure exactly what I'd need later on, but if I only need the mind, yeah this can be changed